metricbeat/magefile.go

```diff
@@ -168,15 +186,13 @@ func IntegTest() {
 // Use TEST_TAGS=tag1,tag2 to add additional build tags.
 // Use MODULE=module to run only tests for `module`.
 func GoIntegTest(ctx context.Context) error {
-	mg.Deps(Fields)
+	if !devtools.IsInIntegTestEnv() {
+		mg.SerialDeps(Fields, Dashboards)
+	}
 	return devtools.GoTestIntegrationForModule(ctx)
 }
 
-// PythonIntegTest executes the python system tests in the integration
```

Comment:
This might have been a merge issue. I think we want to keep this godoc.

Reply:
Fixed.